### ISSUE TITLE
As a member of the dev team, I want to generate load based on file downloads

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#4487]
After standup this morning @kcondon and I chatted about load/performance testing in general and generating load based on file downloads specifically.

The exact scope of the pull request I'll work on is still somewhat undefined here are some thoughts

- http://guides.dataverse.org/en/4.8.5/developers/testing.html#load-performance-testing characterizes "Load/Performance Testing" as "Future Work" but we can better document the sorts of things we do in practice.
- A simple script could be written that takes in a file with a list of IDs as input and downloads the files one by one (blocking). The URL of the Dataverse installation needs to be configurable (default to localhost:8080). Add a option to sleep between requests (default to 0 miliseconds). Provide an option like curl's `--insecure` to bypass a check for a valid cert. Send file to /dev/null. Would be nice to be able to supply an API token.
- Extend script above to download files in parallel (add `&` to background for a bash script). An option to control how many parallel threads to run at once would be nice.
- Copy the important scripts, docs, and config from https://github.com/IQSS/dataverse-helper-scripts/tree/master/src/stress_tests into the main repo (dataverse_tasks.py, creds_template.json, basic_test_04_mydata.py, etc.)

**Comment by pdurbin** [comment]
I just created pull request #4488 to implement the simple script described above and some doc improvements. I'm putting this in Code Review at https://waffle.io/IQSS/dataverse to get feedback from @landreev and others who may want to weigh in on the approach and so @kcondon and try the simple script and give me feedback.

**Comment by pdurbin** [comment]
@scolapasta I pushed b3faa9f to explain that the scripts at `tests/locust` are not currently used (by @kcondon anyway). Perhaps the three of us should get together to discuss the definition of done for this issue.

**Comment by landreev** [comment]
I moved it into QA; 
@kcondon: note that you can use the same script, with the same file.txt to generate a swarm of simultaneous downloads - by adding the "&" on the curl command line: 
```
curl --insecure --silent --show-error --output /dev/null $BASE_URL/api/access/datafile/$i &
```

this will start all the downloads essentially at once.
